ci_framework/roles/reproducer/tasks/ci_job.yml

```diff
@@ -71,19 +92,31 @@
           }}
         mode: "0644"
 
-    - name: Copy environment files to controller node
+    - name: CI reproducer dedicated tasks
       tags:
         - bootstrap
-      ansible.builtin.copy:
-        src: "{{ reproducer_basedir }}/parameters/"
-        dest: "./{{ job_id }}-params"
+      when:
+        - cifmw_job_uri is defined
+      block:
+        - name: Copy environment files to controller node
+          ansible.builtin.copy:
+            src: "{{ _reproducer_basedir }}/parameters/"
+            dest: "./{{ job_id }}-params"
 
-    - name: Inject reproducer dedicated parameter file
-      tags:
-        - bootstrap
-      ansible.builtin.template:
-        src: "reproducer_params.yml.j2"
-        dest: "./{{ job_id }}-params/reproducer_params.yml"
+        - name: Inject reproducer dedicated parameter file
+          ansible.builtin.template:
+            src: "reproducer_params.yml.j2"
+            dest: "./{{ job_id }}-params/reproducer_params.yml"
+
+        - name: Generate CI job playbook
+          ansible.builtin.template:
+            dest: "src/github.com/openstack-k8s-operators/ci-framework/{{ job_id }}_play.yml"
```

Comment:
Minor thing: `src` and `dest` is reversed. Other tasks have `src/dest` this one has `dest/src`

Reply:
Probably it was like that before.